source/common/http/http1/codec_impl.h

```diff
@@ -281,18 +293,22 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   virtual int onHeadersComplete() PURE;
 
   /**
-   * Called when body data is received.
-   * @param data supplies the start address.
-   * @param length supplies the length.
+   * Called when body data is available for processing.
```

Comment:
how about some comments here or on our buffered data when we buffer and when we process, and why we do it this way?

Reply:
Done.